feature_engineering/update_dataset.py

```diff
@@ -3,19 +3,22 @@
 '''
 import pathlib
 import random
+import re
 import sys
 import os
 import shutil
 import time
 import numpy as np
 import pandas as pd
 import tqdm
+import json
+from machine_learning.cnn.image_dataset_generator import PairWriter
```

Comment:
What does the PairWriter do?  I can't seem to find the PairWriter code that is being imported here.

Reply:
It writes frame pairs matched by VideoAssetProcessor to disk. Moved the class to file.